### ISSUE TITLE
Checking against hardcoded values in `getCountryDropDownList`

### ISSUE THREAD

**Issue by johanneslumpe** [mukeshsoni/react-telephone-input#28]
Not sure whether you really intended to do the following:

```javascript
let itemClasses = classNames({
    country: true,
    preferred: country.iso2 === 'us' || country.iso2 === 'gb',
    active: country.iso2 === 'us',
    highlight: this.state.highlightCountryIndex === index
});
```

On first sight it does seem like that this is not supposed to be like this. Why should only the `us` and `gb` codes determine whether something is preferred?

**Comment by mukeshsoni** [comment]
No that's a bug. Thanks for reporting

**Comment by mukeshsoni** [comment]
Fixed in 3.6.0 release
